contracts/apps/disputable/IArbitrable.sol

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Not really sure where this and the `IArbitrator` interfaces live best, but perhaps we should mention that it's used by the Court as well?

I wonder if we should eventually maintain some sort of `@aragon/contract-interfaces` package that holds things like this?

Reply:
I think it's fine to have these here for now

Reply:
Cool; yeah, for now maybe we can add a README to every `lib/` subdirectory to note where they come from or etc. so there's some context (especially as we don't follow 792 😉)